frontend/mulighetsrommet-veileder-flate/src/hooks/useInitializeArbeidsmarkedstiltakFilterForBruker.ts

```diff
@@ -12,19 +17,25 @@ export function useInitializeArbeidsmarkedstiltakFilterForBruker() {
   const brukersInnsatsgruppe = innsatsgrupper?.find(
     (gruppe) => gruppe.nokkel === brukerdata?.innsatsgruppe,
   );
+  const relevanteEnheterForBruker = relevanteEnheter(brukerdata);
```

Comment:
Lurer også på om vi kunne pushet all denne business-logikken fra fontend til backend. 

Hva med et endepunkt for `getDefualtArbeidsmarkedstiltakFilterForBruker` som henter innsatsgrupper, enheter osv. Kan hende det kunne erstattet behovet for en del av feltene vi har i `Brukerdata`-modellen vi har nå :thinking:

Reply:
Endret brukerdata modellen litt nå